pkg/state/appender.go

```diff
@@ -408,18 +421,24 @@ type appendTxParams struct {
 	invokeExpressionActivated        bool // TODO: check feature naming
 	validatingUtx                    bool // if validatingUtx == false then chans MUST be initialized with non nil value
 	stateActionsCounterInBlock       *proto.StateActionsCounter
+	currentMinerPK                   crypto.PublicKey
+
+	snapshotGenerator *snapshotGenerator
```

Comment:
Because txAppender is initialized in `newStateManager` and snapshotApplier for example needs `checkerInfo` which is unique for each block, as well as stateActionsCounter. Are suggesting adding "snapshotApplierInfo" and initializing it in txPerformer?